### PR TITLE
Updated iOS Radio button & checkbox documentation.

### PR DESCRIPTION
**Pull request by omoralesvivo**
Ticket: https://thumbtack.atlassian.net/browse/DS-989

WHY:

Updates in radio button and checkbox iOS feature set and API require corresponding updates in documentation.

WHAT:

Updated docs to reflect Thumbprint changes:
- Dropped error state for checkbox & radio buttons (not in use nor expected to be in use in the future)
- Renamed labeled versions of the controls to `LabeledRadio` and `LabeledCheckbox`
- `Radio` is now equivalent to `Checkbox`, just the graphical control.
- `LabeledRadio` is feature equivalent to `LabeledCheckbox`, specifically it allows for leading/trailing content placement.

TESTING:

Best I can tell the syntax of the changes is correct. Unaware of how to test the fully functional pages.

### CODE REVIEW

File: www/src/pages/components/radio/ios/index.mdx [hunk 1]
```diff
@@ -3,31 +3,119 @@ title: Radio
 description: Control for selecting an option inside a form
 ---
 
+import { graphql } from 'gatsby';
+import { ComponentHeader, ComponentFooter } from 'components/thumbprint-components';
+
+<ComponentHeader data={props.data} />
```
Comment (kevinmbeaulieu) [inline www/src/pages/components/radio/ios/index.mdx:9]: Good catch 🙇

File: www/src/pages/components/radio/ios/index.mdx [hunk 2]
```diff
@@ -3,31 +3,119 @@ title: Radio
 description: Control for selecting an option inside a form
```
Comment (kevinmbeaulieu) [inline www/src/pages/components/radio/ios/index.mdx:3]: nit: Could you also change this description to "Boxes for checking and unchecking single values in forms." for consistency with the other platforms while you're here?

**Review by kevinmbeaulieu — DISMISSED** [review]
For future reference, you can click the Details link next to the deploy/netlify CI check on github to view a preview of your changes

Screen Shot 2021-07-27 at 1 04 41 PM

File: www/src/pages/components/radio/ios/index.mdx (hunk 2 above)
Comment (omoralesvivo) [reply www/src/pages/components/radio/ios/index.mdx]: I'm not sure that's the best description we could have but I got it in for consistency.

**Comment by omoralesvivo** [comment]
I was already looking at the formatted results but there were some links that wouldn't work from GitHub.

**Review by kevinmbeaulieu — APPROVED** [review]

**Pull request merged**